frontend-project/src/pages/cases/new/index.tsx

```diff
@@ -142,6 +156,7 @@ const CasesNewForm: FunctionComponent<CasesNewFormProps> = ({ tags, users, dispa
               <Form.Item
                 label={formatMessage({ id: 'cases-new.form.notified-users.label' })}
                 name="notified-users"
+                Option
```

Comment:
what does this `Option` in `Form.Item` do?

Reply:
jak dobrze rozumiem https://ant.design/components/select/ to w tym przypadku po prostu za możliwość wielokrotnego wyboru userów

Reply:
Hmm musiałam to przez przypadek zakomitować. Dzięki za wyłapanie.